cns/configuration/cns_config.json

```diff
@@ -32,5 +32,9 @@
     "MellanoxMonitorIntervalSecs": 30,
     "AZRSettings": {
         "PopulateHomeAzCacheRetryIntervalSecs": 60
+    },
+    "GRPCSettings": {
+        "IPAddress": "localhost",
+        "Port": 8080
```

Comment:
Since these changes are just for a prototype right now, I don't think we should add placeholder values for these settings to the `cns_config.json`. If someone wants to run CNS with this feature enabled they can add these settings in the config themselves.